Automattic/Gravatar-SDK-Android
# Fix: ProfileSummary vertical alignment

Closes #202

### Description

The `ProfileSummary` component has a wrong alignment of the name/location/button combo. Here's the visual comparison:
| Expected | Actual | 
|---|---|
| <img width="451" alt="Screenshot 2024-07-02 at 10 51 25" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/12660801/a94530dd-d404-43d4-bb14-0ee89f019454"> | <img width="558" alt="Screenshot 2024-07-02 at 10 51 54" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/12660801/f373d5af-5a15-4c31-b760-9251fa670ba0"> |

This PR updates it to follow the design:

| Location | Without location |
|---|---|
| <img width="456" alt="Screenshot 2024-07-02 at 12 12 10" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/12660801/af6d0ba5-909a-41b2-af33-f3f75d76b28e"> | <img width="452" alt="Screenshot 2024-07-02 at 12 12 21" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/12660801/59067e5a-b078-4984-b037-09d4e7b0f484"> |

### Testing Steps

There's a new test screenshot to cover that scenario but if you want to test it yourself here are the steps:
1. Go to gravatar.com and remove the location from your profile
2. Open the demo app
3. Go to the profile tab and fetch your profile
4. Confirm the UI looks as it should - text composables are center aligned
5. Go to gravatar.com and add the location to your profile 
6. Back to the demo app
7. Go to the profile tab and fetch your profile
8. Confirm the UI looks as it should
